Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -72,6 +76,16 @@ public Initializer(string id, string tokenServerUrl) : base(tokenServerUrl)
                 Scopes = new List<string>();
             }
 
+            /// <summary>Extracts a <see cref="Key"/> from the clientCredentialParameters which is obtained from parsing the Json Key.</summary>
+            public Initializer FromJsonKey(ClientCredentialParameters clientCredentialParameters)

```

Comment:
btw, this would probably mean renaming the method to something more matching like FromPkcs8()  FromPkcs8Key or FromPkcs8PrivateKey, because FromJsonKey(string pkcs8PrivateKey doesnt make much sense anymore).